gazelle/js/typescript/tsconfig.go

```diff
@@ -44,7 +61,9 @@ var DefaultConfigPaths = TsConfigPaths{
 	Map: &map[string][]string{},
 }
 
-var Log = logrus.New()
+// Matches strings starting with ../, ./, @ or .
+// Used to detect if a path could possibly expanded using baseUrl
+var baseurl_validation_regex = regexp.MustCompile(`^(\.\.?\/|@|\.)`)
```

Comment:
Do you have an example where the `@` case is required? I thought that would be considered a named import so it wouldn't be relative?

Reply:
Actually you are right the @ should be omitted. My thought was the @ pattern is mostly used by path mappings which are handled before so we could save us a case here, but there also could be folders starting with @ which are not directly relative, where this regex is the condition guard.

If I remove the @ case a regex will be might be overkill wouldn't it? The cases are only `.,./,../` which also could be handled with an if and strings.HasPrefix. Any opinions on what would be the preferred way?

Reply:
I'd say put it in a util method and then do whatever you want, can always change the method later without the caller caring. Note that I think the `^\.` will also catch basic `.dotfile` cases? Maybe:
```
func isRelative(s string) {
   return s == "." or strings.HasPrefix(s, "./") or strings.HasPrefix(s, "../")
}
```
?

Reply:
Could also put the `path.IsAbs` within the util method, although I think that isn't actually needed?

Reply:
I put it with inside, because I think technically it is part of the check and so it is all tied together